.github/workflows/build-reusable.yaml

```diff
@@ -67,6 +108,13 @@ jobs:
             -DinstallAtEnd=true \
             clean install
 
+      - name: Clean up Develocity
+        if: always()
+        shell: bash
+        run: |
+          rm -f .mvn/extensions.xml
+          echo "MAVEN_OPTS=" >> "$GITHUB_ENV"
```

Comment:
Why do we truncate the `MAVEN_OPTS`?

Reply:
The `gradle/develocity-actions/maven-setup` action modifies the `MAVEN_OPTS` environment variable. We restore it to the default (_empty_) value, to prevent site generation from failing.

I added some comments and checks in https://github.com/apache/logging-parent/pull/229/commits/8975e12ce43007e9816e9996363e63e87c3ee477 and https://github.com/apache/logging-parent/pull/229/commits/5980ad9cc94f85718edef8f720eea50b49575696